src/entities/pool.ts

```diff
@@ -66,17 +68,20 @@ export class Pool {
    * @param liquidity The current value of in range liquidity
    * @param tickCurrent The current tick of the pool
    * @param ticks The current state of the pool ticks or a data provider that can return tick data
+   * @param tickSpacing The spacing between ticks
```

Comment:
Clarify that if `tickSpacing` is undefined, it will default to `TICK_SPACINGS[fee]`, and this only works for UniV3 and PCSV3.